website_sale_product_minimal_price/models/product_template.py

```diff
@@ -29,21 +50,28 @@ def _get_combination_info(
             combination=combination, product_id=product_id, add_qty=add_qty,
             pricelist=pricelist, parent_combination=parent_combination,
             only_template=only_template)
-
         if (only_template and self.env.context.get('website_id') and
                 self.product_variant_count > 1):
-            cheaper_variant = self.product_variant_ids.sorted(
-                key=lambda p: p.website_price)[:1]
-
-            res = cheaper_variant._get_combination_info_variant()
-
+            partner = self.env.context.get('partner')
+            pricelist_id = self.env.context.get('pricelist')
+            minimal_price = self._get_minimal_price(
+                pricelist_id, add_qty, partner)
             combination_info.update({
-                'price': res.get('price'),
-                'list_price': res.get('list_price'),
-                'has_discounted_price': res.get('has_discounted_price'),
+                'price': minimal_price[0][1],
+                'list_price': minimal_price[0][1],
             })
         return combination_info
 
+    # TODO: Active cache
+    @ormcache('self.id', 'product_attribute_values')
+    def _get_cheaper_combination(self, product_attribute_values):
```

Comment:
I suggest to use the `_ids` suffix, to indicate it's a list of ids and not a recordset.

```suggestion
    def _get_cheaper_combination(self, product_attribute_value_ids):
```